fmriprep/cli/run.py

```diff
@@ -92,6 +92,11 @@ def get_parser():
         choices=['MNI152NLin2009cAsym'], default='MNI152NLin2009cAsym',
         help='volume template space (default: MNI152NLin2009cAsym)')
 
+    g_conf.add_argument(
+        '--output-grid-reference', required=False, action='store', default=None,
+        help='Grid reference image for resampling BOLD files to volume template space. '
+             'It determines the grid (i.e., affine matrix) of the output files if provided.')
```

Comment:
I would mention that this "determines the field of view and resolution of the output images, but is not used in normalization".